### DISCUSSION TITLE
【应用市场】功能增强，提升使用体验

### DISCUSSION THREAD

**Discussion by z-hanzhe** [halo-dev/halo#5112] [Ideas]
### 你当前使用的版本

2.11.2

### 描述一下此特性

1. 现在搜索插件严格区分大小写，感觉没必要希望这里改成忽略大小写
2. 有些插件起名比较文艺，具体作用都在插件描述中，而现在的应用市场仅支持搜索标题，希望优化一下
3. 应用市场提供的所有插件，并不能很清晰的分辨是官方还是第三方开发的
4. 希望插件描述中提供的图片可以点击查看大图
5. 推荐多加一些过滤条件，例如：
    - 只看官方插件
    - 只看该作者的插件
    - 只看兼容当前Halo版本的插件
    - 后续想到我在来补充......

**Comment by ruibaby** [comment]
感谢反馈问题，但应用市场的问题应该不属于 Halo 本身的仓库，所以我将转移到 Discussion。
